### PR TITLE
[Homework #6] - JS-Basics-1 by Usmonbek Rustamov

### PR DESCRIPTION
**Pull request by usmonbek-umarbekovich**
Hello,

Solved all questions except task 8 (There was no question with that link)

### CODE REVIEW

File: homeworks/usmonbek-rustamov/06-js-basics-1/script.js [hunk 1]
```diff
@@ -0,0 +1,276 @@
+// Task 1: https://www.codewars.com/kata/56dec885c54a926dcd001095/train/javascript
+function opposite(number) {
+  return -number;
+}
+
+// Task 2: https://www.codewars.com/kata/57356c55867b9b7a60000bd7/train/javascript
+function basicOp(operation, value1, value2) {
+  return eval(value1 + operation + value2);
```
Comment (VitalikMelnyk) [inline homeworks/usmonbek-rustamov/06-js-basics-1/script.js:8]: In future homeworks, avoid using `eval` function.
https://stackoverflow.com/questions/86513/why-is-using-the-javascript-eval-function-a-bad-idea

File: homeworks/usmonbek-rustamov/06-js-basics-1/script.js [hunk 2]
```diff
@@ -0,0 +1,276 @@
...
+    }
+  }
+
+  return [];
+}
+
+// Task 17: https://www.codewars.com/kata/576757b1df89ecf5bd00073b/train/javascript
+function towerBuilder(nFloors) {
+  const tower = [];
+  const maxStars = 2 * nFloors - 1;
```
Comment (VitalikMelnyk) [inline homeworks/usmonbek-rustamov/06-js-basics-1/script.js:232]: Redundant variable.

Comment (usmonbek-umarbekovich) [reply homeworks/usmonbek-rustamov/06-js-basics-1/script.js]: I was using different approach at first and `maxStars` was a variable being used. Then I changed my approach, but forgot to remove it

**Review by VitalikMelnyk — APPROVED** [review]
Well Done @usmonbek-umarbekovich!
Approved.

**Pull request merged**